changelogs/fragments/76-lint-changelog-yaml.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - "Add ``changelogs/changelog.yaml`` file format linting subcommand that was previously part of antsibull-lint (https://github.com/ansible-community/antsibull-changelog/pull/76)."
```

Comment:
Might be good to also link to the associated discussion you opened. unless you don't want that in the changelog.

Reply:
Done.